Commands/Referal/GetRefInfoCommand.cs

```diff
@@ -0,0 +1,11 @@
+namespace hypixel
+{
+    public class GetRefInfoCommand : Command
+    {
+        public override void Execute(MessageData data)
+        {
+            var refInfo = ReferalService.Instance.GetReferalInfo(data.User);
+            data.SendBack(data.Create("refInfo", refInfo));
```

Comment:
in the documentation the command is called getRefInfo. Am i wrong?

Reply:
correct, `refInfo`  is the response command to `getRefInfo`

Reply:
since this is the only commend I am merging